lib/flowDiagnostics.js

```diff
@@ -62,43 +62,56 @@ async function getDiagnostics(document) {
 			const {level, messageComponents} = message
 			if (!messageComponents.length) return
 
-			const
-				[baseMessage, ...other] = messageComponents,
-				range = baseMessage.range;
-
-			if (range == null) return;
-
-			const file = range.file
-
-			let diag = {
-				severity: level,
-				startLine: range.start.line,
-				startCol: range.start.column,
-				endLine: range.end.line,
-				endCol: range.end.column,
-				msg: '',
-			}
-
-			let details = [];
-			other.forEach(part => {
-				let partMsg = part.descr;
-				if (partMsg && partMsg !== 'null' && partMsg !== 'undefined') {
```

Comment:
The error reporter in Nuclide doesn't have this check so I think it's safe to remove it.